project/PlayCrossCompilation.scala

```diff
@@ -1,4 +1,4 @@
 import uk.gov.hmrc.playcrosscompilation.AbstractPlayCrossCompilation
-import uk.gov.hmrc.playcrosscompilation.PlayVersion.Play26
+import uk.gov.hmrc.playcrosscompilation.PlayVersion.Play28
 
-object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play26)
\ No newline at end of file
+object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play28)
```

Comment:
Suggestion: I would keep the default as 2.6, reduce the risk of adding in non-backwards compatible features when developing.

Reply:
No problem. Corrected.